### PR TITLE
Add postprocessor

### LINKED ISSUE #6: MathJax still present in file
Thank you for this :gem:, much appreciated :clap:

I noticed that MathJax is still referenced in the file. When doing server side math rendering I would not have expected to see the client side rendering libraries loaded at all.

Using a similar example to that from @cirosantilli:

```
= bla
:stem: latexmath
:docinfo: shared

[latexmath]
++++
\sqrt{1+1} = 2
++++
```

At the end of the html the following is still including which is not what I had expected:

```html
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  messageStyle: "none",
  tex2jax: {
    inlineMath: [["\\(", "\\)"]],
    displayMath: [["\\[", "\\]"]],
    ignoreClass: "nostem|nolatexmath"
  },
  asciimath2jax: {
    delimiters: [["\\$", "\\$"]],
    ignoreClass: "nostem|noasciimath"
  },
  TeX: { equationNumbers: { autoNumber: "none" } }
})
MathJax.Hub.Register.StartupHook("AsciiMath Jax Ready", function () {
  MathJax.InputJax.AsciiMath.postfilterHooks.Add(function (data, node) {
    if ((node = data.script.parentNode) && (node = node.parentNode) && node.classList.contains('stemblock')) {
      data.math.root.display = "block"
    }
    return data
  })
})
</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_HTMLorMML"></script>
```

### PR DESCRIPTION
**Pull request by tani**
Closes #6. This postprocessor replaces MathJax with the empty string. Cheers.

### CODE REVIEW

**Comment by jirutka** [comment]
Thanks! Can you please add an integration test(s) for this?

File: lib/asciidoctor/katex/postprocessor.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require 'asciidoctor' unless RUBY_PLATFORM == 'opal'
+require 'asciidoctor/extensions' unless RUBY_PLATFORM == 'opal'
+
+module Asciidoctor::Katex
+  class Postprocessor < Asciidoctor::Extensions::Postprocessor
+    def process document, output
```
Comment (jirutka) [inline lib/asciidoctor/katex/postprocessor.rb]:
```diff
-    def process document, output
+    def process(document, output)
```

**Comment by tani** [comment]
I added a test to check the existence of MathJax scripts. Please check it. Thanks.

**Comment by jirutka** [comment]
Merged with a few changes. Thanks for your contribution!

**Comment by tani** [comment]
Thanks for the great extension!

**Pull request merged**